index.d.ts

```diff
@@ -551,6 +556,8 @@ type Execa$<StdoutStderrType = string> = {
 
 	The `shell` option must be used if the `command` uses shell-specific features (for example, `&&` or `||`), as opposed to being a simple `file` followed by its `arguments`.
 
+	As a convenience, the result from previous `` $`command` `` or `` $.sync`command` `` calls can be used as template expressions in subsequent commands. See the example below `` with results from `$` or `$.sync` `` for more details.
```

Comment:
We might want to be explicit that `stdout` (not `stderr`) is being used.

Reply:
Good call 👍 How's this: https://github.com/sindresorhus/execa/pull/528/commits/bc64405c7e9f44d5b42571d4b699b0426c7897b1